### DISCUSSION TITLE
Question about  <"configurating a New robot">

### DISCUSSION THREAD

**Discussion by xjmpeter** [NVlabs/curobo#32] [Software Q&A]
Hi,

Thank you for your great code!

I followed the  instructions.
When I run the script "omni_python   .... motion_gen_reacher.py --robot  myrobot.yml --visualize_spheres"   , my robot is popup on isaac-sim .
After click the "play" button, and then move the red cube , but robot doesn't move.
There is no error ,only some warning
The log is as follow:
![image](https://github.com/NVlabs/curobo/assets/38111315/46f025a6-3cd2-4d74-81fa-1d4bc6c48e53)

**Comment by balakumar-s** [comment]
Does the ik example or mpc example work for your robot?

**Comment by peter05010402** [comment]
@xjmpeter  
Do you solve the problem?
I also face the problem.

**Comment by balakumar-s** [comment]
See if this solves the problem:

https://github.com/NVlabs/curobo/issues/16#issuecomment-1798939585

**Comment by peter05010402** [comment]
Hi @balakumar-s 

mpc_example is work for my robot. 
But the end of robot move up while the target cube down.
motion_gen_reacher doesn't work, print warning: “Plan did not converge to a solution.  No action is being taken.”

**Reply by balakumar-s** [reply]
Did you setup the self collision ignore correctly? It looks like the robot might have self collisions at the start state.

**Comment by peter05010402** [comment]
when Run  "omni_python curobo/examples/isaac_sim/motion_gen_reacher.py --robot /{PATH}/ur5e.yml --visualize_spheres",
there is no links' spheres.

**Reply by balakumar-s** [reply]
What value did you set {PATH} to?
